orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -52,6 +54,9 @@ class DependentPipelineStarter implements ApplicationContextAware {
   @Autowired(required = false)
   ArtifactResolver artifactResolver
 
+  @Autowired
+  Registry registry
```

Comment:
Worth changing to an autowired constructor?

Reply:
Can you point me to an example of someplace where we have autowired constructor with non required beans where we correctly fail to start up when things are missing that shouldn't be?

Reply:
Thx <3